sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -169,6 +172,69 @@ def test_do_with_side_outputs_maintains_unique_name(self):
     pipeline.run()
 
   @attr('ValidatesRunner')
+  def test_read_from_text_metrics(self):
+    from apache_beam.io import iobase
+    from apache_beam.io.range_trackers import OffsetRangeTracker
+    class CountingSource(iobase.BoundedSource):
```

Comment:
Can you re-use the CountingSource available in following location ?

https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py#L606